### ISSUE TITLE
Bug (typo): change "user" parameter to "username" in /docs/infomaniak.md 

### ISSUE THREAD

**Issue by alexander-t7** [qdm12/ddns-updater#165]
**TLDR**: Correction to your documentation in [/docs/infomaniak.md](../blob/master/docs/infomaniak.md) --> should read "username" not "user" under compulsory parameters.

1. Is this urgent: No
2. DNS provider(s) you use: Infomaniak
3. Program version: "Running version latest built on 2021-01-30T20:01:15Z (commit 78c86b0)"
4. What are you using to run the container: docker-compose

Logs:

```log
2021-02-06T16:36:30.623257289Z 2021-02-06T17:36:30.623+0100   ERROR   empty username
```

Configuration file (**remove your credentials!**):

```json
{
  "settings": [
    {
      "provider": "infomaniak",
      "domain": "REDACTED",
      "host": "@",
      "user": "REDACTED",
      "password": "REDACTED",
      "ip_version": "ipv4"
    }
  ]
}
```

Host OS: Ubuntu 20.04.2 LTS

First of all, thanks @qdm12 for this container! It's brilliant and fits my needs perfectly.
This is not a bug, just to mention a typo for you to correct in [/docs/infomaniak.md](../blob/master/docs/infomaniak.md) ... originally I ran the container using the "user" string in my ```json.config``` but was confronted with the single error line in the log mentioned above. After a quick look at [/internal/settings/infomaniak.go](../blob/master/internal/settings/infomaniak.go) it looks like go was indeed looking for a "username" string rather than "user", so I corrected this line in my config, and then it ran as expected.

```json
"username": "REDACTED",
```

Thanks for your hard work!

**Comment by fredericrous** [comment]
Thank you for the report @alexander-t7 . I fixed it directly on master @qdm12 as this was a minor documentation change and had been reviewed by alexander and me
